task/control-plane-agent/src/mgs_common.rs

```diff
@@ -47,14 +45,8 @@ impl MgsCommon {
     }
 
     pub(crate) fn identity(&self) -> VpdIdentity {
-        if let Some(identity) = *self.identity.borrow() {
-            return identity;
-        }
-
-        let id = identity();
-        let mut cached = self.identity.borrow_mut();
-        *cached = Some(id);
-        id
+        let packrat = Packrat::from(PACKRAT.get_task_id());
```

Comment:
What prevents us from talking to `packrat` before it's ready?  Are we waiting on `jefe` state somewhere in `control-plane-agent` startup?

Reply:
This might be too roundabout for comfort, but: we only get here if we received a packet on the network, and `net` waits for `jefe` to populate packrat (by waiting for the transition to A2).

Reply:
:+1: Added in 52ac248c